### PR TITLE
[KeyVault] Shifted the location of key CVM release policy to GitHub

### PR DESCRIPTION
**Pull request by BethanyZhou**
## Description
* Shifted the location of key CVM release policy to GitHub
* Added fallback logic if fetching cvm release policy from GitHub failed

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/KeyVault/KeyVault/Commands/Key/AddAzureKeyVaultKey.cs [hunk 1]
```diff
@@ -629,5 +628,35 @@ internal Track2Sdk.JsonWebKey CreateTrack2WebKeyFromFile()
...
+                    {
+                        defaultCVMPolicy = reader.ReadToEnd();
+                    }
+                }
+                catch(Exception ex)
+                {
+                    WriteWarning(string.Format(Resources.FetchDefaultCVMPolicyFailed, ex.Message));
+                };
+            }
+            return defaultCVMPolicy;
```
Comment (isra-fel) [inline src/KeyVault/KeyVault/Commands/Key/AddAzureKeyVaultKey.cs:659]: What will happen if return null?

File: src/KeyVault/KeyVault/Resources/skr-policy.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
+﻿{
```
Comment (isra-fel) [inline src/KeyVault/KeyVault/Resources/skr-policy.json:1]: We should make sure that CVM team can reach out to us when the default policy file needs to be updated.

**Review by isra-fel — DISMISSED** [review]

File: src/KeyVault/KeyVault/Resources/skr-policy.json (hunk 2 above)
Comment (BethanyZhou) [reply src/KeyVault/KeyVault/Resources/skr-policy.json:1]: emailed to CVM team

File: src/KeyVault/KeyVault/Commands/Key/AddAzureKeyVaultKey.cs (hunk 1 above)
Comment (BethanyZhou) [reply src/KeyVault/KeyVault/Commands/Key/AddAzureKeyVaultKey.cs:652]: throw exception

**Pull request merged**